internal/daemon/dpusidemanager_test.go

```diff
@@ -69,9 +69,10 @@ var _ = g.Describe("DPU side maanger", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
```

Comment:
iiuc, you're making the DpuOperatorConfig be the owner for the vsp directly, not chaining through the Daemon because that won't work?

Reply:
Yeah exactly. I tried chaining through the daemon first, but after some trial / error and reading up on the docs, it seems kubernetes does not allow pods or daemonsets to own other controllers. It has to be a higher level resource (i.e. a custom resource).